shared/libretime_shared/config/_models.py

```diff
@@ -197,6 +197,31 @@ class AudioOpus(BaseAudio):
     format: Literal[AudioFormat.OPUS] = AudioFormat.OPUS
 
 
+class HlsStream(BaseModel):
+    fragment_prefix: str = "mp3_128"
+    bitrate: str = "128k"
+    sample_rate: int = 44100
+    codec: str = "libmp3lame"
+
+
+class HlsOutput(BaseModel):
+    kind: Literal["hls"] = "hls"
+    enabled: bool = False
+    public_url: Optional[AnyUrl] = None
```

Comment:
If it isn't used, feel free to remove

Reply:
In the StreamSettings.php, public_url is constructed (if not specified) after output specific setup, so it's harmless to let it be: it allows one to explicitly setup public url instead of reconstructing from host:port/mount.
The  mobile settings seams also shared among differents outputs, but i haven't understand its usage.
My guess is to let them just in case